awx/main/models/ha.py

```diff
@@ -160,7 +160,7 @@ def __str__(self):
         default=100,
         editable=False,
     )
-    capacity_adjustment = models.DecimalField(default=Decimal(1.0), max_digits=3, decimal_places=2, validators=[MinValueValidator(Decimal(0.0))])
+    capacity_adjustment = models.DecimalField(default=Decimal(0.75), max_digits=3, decimal_places=2, validators=[MinValueValidator(Decimal(0.0))])
```

Comment:
The default is tracked in migrations

https://github.com/ansible/awx/blob/01eb162378df85edbdb65ab5922fd3c54573e75e/awx/main/migrations/0049_v330_validate_instance_capacity_adjustment.py#L19

So your change would require adding a new migration file. For temporary administrative reasons you are sure to be unaware of, this is going to be harder than you're thinking.

Reply:
@AlanCoding I have added a migration file for it.